### PR TITLE
Print error message in server when adding an existing player

### PR DESCRIPTION
**Pull request by rollandf**
Solves #148 
Here an example of the output :
add player: Random Driver lane: 0 car: 3
Error handling message: Player exists: Random Driver

### CODE REVIEW

File: rose/server/game.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ def stop(self):
 
     def add_player(self, name):
         if name in self.players:
+            print 'Failed to add player:', name, 'player already exists'
```
Comment (nirs) [inline rose/server/game.py]: This is not the place to fix this - please print the exception in lineReceived, so see all errors, not only this specific error.

File: rose/server/game.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def stop(self):
...
             raise error.PlayerExists(name)
```
Comment (nirs) [inline rose/server/game.py]: We should make sure this error has clear text when printed, so we can do later:

    except Exception as e:
        print "Error handling message: %s" % e

**Comment by rollandf** [comment]
@nirs can you review?

File: rose/server/game.py [hunk 3]
```diff
@@ -56,6 +56,7 @@ def stop(self):
...
         if not self.free_cars:
             raise error.TooManyPlayers()
```
Comment (nirs) [inline rose/server/game.py]: Same - we want generic way to show all errors on the server side.

**Comment by rollandf** [comment]
Here an example of the server output when adding an existing player:

add player: Random Driver lane: 0 car: 3
Error handling message: Player exists: Random Driver

**Comment by nirs** [comment]
:+1:

**Pull request merged**